snappi_ixnetwork/trafficitem.py

```diff
@@ -1129,6 +1130,22 @@ def _configure_size(self, ce_dict, size):
                 ce["frameSize"]["incrementFrom"] = size.increment.start
                 ce["frameSize"]["incrementTo"] = size.increment.end
                 ce["frameSize"]["incrementStep"] = size.increment.step
+            elif size.choice == "weight_pairs":
+                if size.weight_pairs.choice == "predefined":
+                    ce["frameSize"]["type"] = "presetDistribution"
+                    if size.weight_pairs.predefined == "ipv6_imix":
+                        ce["frameSize"]["presetDistribution"] = "ipV6Imix"
+                    elif size.weight_pairs.predefined == "standard_imix":
+                        ce["frameSize"]["presetDistribution"] = "standardImix"
+                    elif size.weight_pairs.predefined == "tcp_imix":
+                        ce["frameSize"]["presetDistribution"] = "tcpImix"
+                    elif size.weight_pairs.predefined == "ipsec_imix":
+                        ce["frameSize"]["presetDistribution"] = "ipSecImix"    
+                    else :
+                        ce["frameSize"]["presetDistribution"] = size.weight_pairs.predefined
```

Comment:
is it awes true to assigned `size.weight_pairs.predefined` directly to `presetDistribution`. I feel that string should support in IxNetwork. Please validate.